JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -24,335 +24,357 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
+.bainIndependentSamplesDescriptivesTable <- function(dataset, options, bainContainer, ready, position) {
 
-  if (!is.null(jaspResults[["descriptivesTable"]])) return() #The options for this table didn't change so we don't need to rebuild it
-		if (options[["descriptives"]]) {
+	if (!is.null(bainContainer[["descriptivesTable"]]) || !options[["descriptives"]]) return() 
 
-	  descriptivesTable <- createJaspTable("Descriptive Statistics")
-	  descriptivesTable$dependOn(options =c("variables", "descriptives", "descriptivesPlotsCredibleInterval", "groupingVariable"))
-		descriptivesTable$position <- position
+	descriptivesTable <- createJaspTable("Descriptive Statistics")
+	descriptivesTable$dependOn(options =c("variables", "descriptives", "credibleInterval"))
+	descriptivesTable$position <- position
 
-	  descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
-		descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
-	  descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
-	  descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number")
-	  descriptivesTable$addColumnInfo(name="sd",                   title = "SD", type="number")
-	  descriptivesTable$addColumnInfo(name="se",                   title = "SE", type="number")
+	descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
+	descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
+	descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
+	descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number")
+	descriptivesTable$addColumnInfo(name="sd",                   title = "SD", type="number")
+	descriptivesTable$addColumnInfo(name="se",                   title = "SE", type="number")
 
-		interval <- 100 * options[["descriptivesPlotsCredibleInterval"]]
-		overTitle <- paste0(interval, "% Credible Interval")
-		descriptivesTable$addColumnInfo(name="lowerCI",              title = "Lower", type="number", overtitle = overTitle)
-	  descriptivesTable$addColumnInfo(name="upperCI",              title = "Upper", type="number", overtitle = overTitle)
-		
-		jaspResults[["descriptivesTable"]] <- descriptivesTable
-
-		if (!ready)
-			return()
-
-		levels <- base::levels(dataset[[ .v(options$groupingVariable) ]])
-		if (length(levels) != 2) {
-			g1 <- "1"
-			g2 <- "2"
-		} else {
-			g1 <- levels[1]
-			g2 <- levels[2]
-		}
+	overTitle <- paste0(100 * options[["credibleInterval"]], "% Credible Interval")
+	descriptivesTable$addColumnInfo(name="lowerCI",              title = "Lower", type="number", overtitle = overTitle)
+	descriptivesTable$addColumnInfo(name="upperCI",              title = "Upper", type="number", overtitle = overTitle)
+	
+	bainContainer[["descriptivesTable"]] <- descriptivesTable
+
+	if (!ready || bainContainer$getError())
```

Comment:
I can't actually find a `setError()` on the container. Is this done in some common file?
If you don't do this anywhere then `bainContainer$getError()` will always be `FALSE`

Reply:
Removed them in the t-test files